packages/remix-react/transition.ts

```diff
@@ -100,7 +100,7 @@ export interface TransitionManagerInit {
 export interface Submission {
   action: string;
   method: string;
-  formData: FormData;
+  formData: FormData | string;
```

Comment:
I think I'd rather have a "json" property instead of re-using formData. FormData is VERY specific in what it is and is not at all JSON.

Reply:
I get that. I was just trying to be minimal in my PR. I'll make the change you requested then.

Reply:
I've updated the Submission interface.